resources/assets/less/layout.less

```diff
@@ -54,6 +54,14 @@ html, body {
   height: 100%;
 }
 
+html {
+  scroll-padding-top: @nav2-height--pinned;
+
+  @media @mobile {
+    scroll-padding-top: @navbar-height;
+  }
+}
+
```

Comment:
I *think* for this to work in safari is change `scroll-padding-top` to `scroll-margin-top`, because currently safari only full support `scroll-margin` based on mdn.

Also if we use `scroll-margin`, we have to use `*` instead of `html`.